egs2/swbd_sentiment/asr1/conf/decode_asr.yaml

```diff
@@ -0,0 +1 @@
+beam_size: 10
```

Comment:
Do we have a sequence of output? If not, `beam_size` to 1 would be fine (just as other slu tasks)